tests/sqllogictests/suits/test.slt

```diff
@@ -0,0 +1,29 @@
+statement ok
```

Comment:
Does this means we need to port all our logic test to `slt`?

Reply:
Just change toriginal file name to .slt and adjust the content syntax

Reply:
'.slt' is strange, can we change it?

Reply:
I think it makes sense to support different file extension in `sqllogictest-rs`, we can submit an issue and implement it.

Reply:
We do not check file extensions. You can use anything as filename :P

_Originally posted by @xxchan in https://github.com/risinglightdb/sqllogictest-rs/issues/118#issuecomment-1342878713_

Reply:
Yes, both are ok, will use `.test`.